tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Upgrade a Running Network
+description: Learn how to perform forkless runtime upgrades on a Polkadot-SDK based blockchain. Add features and modify parameters without network disruption.
+---
+
+# Upgrade a Running Network
+
+## Introduction
+
+One of the key advantages of the Polkadot-SDK development framework is its support for forkless upgrades to the blockchain runtime, which forms the core logic of the chain. Unlike many other blockchains, where introducing new features or improving existing ones often requires a hard fork, Polkadot-SDK enables seamless upgrades even when introducing breaking changes—without disrupting the network's operation.
+
+In traditional blockchain systems, a hard fork is necessary to implement changes, forcing participants to update their software to remain in sync. However, Polkadot-SDK's design incorporates the runtime directly into the blockchain's state, allowing participants to upgrade the runtime by calling the `set_code` function within a transaction. This mechanism ensures that updates are validated using the blockchain's consensus and cryptographic guarantees, allowing runtime logic to be updated or extended without forking the chain or requiring a new blockchain client.
```

Comment:
```suggestion
Polkadot SDK's design incorporates the runtime directly into the blockchain's state, allowing participants to upgrade the runtime by calling the `set_code` function within a transaction. This mechanism ensures that updates are validated using the blockchain's consensus and cryptographic guarantees, allowing runtime logic to be updated or extended without forking the chain or requiring a new blockchain client.
```

I'd remove the first sentence here, just because it's a bit repetitive of what is mentioned in the first paragraph